assets/css/main.css

```diff
@@ -419,3 +419,14 @@ table.results-table .category-row {
     margin-top: 4rem;
     width: 80%;
 }
+
+/*ExamSession management form*/
+.contentDividerExamSession {
```

Comment:
The naming style for CSS classes is `content-divider-exam-session` instead of camelCased `contentDividerExamSession`.

Reply:
Fixed